python/lsst/ts/vent/controller/sequent.py

```diff
@@ -0,0 +1,144 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2020 Sequent Microsystems
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+# Copied from:
+#     https://github.com/SequentMicrosystems/megaind-rpi
+#     https://github.com/SequentMicrosystems/16inpind-rpi
+# Added to the ts_atbuilding_vents repository 21 August 2024, Brian Brondel
+
+__all__ = ["write_channel", "read_channel"]
+
+import smbus3
+
+INPUTS16_DEVICE_ADDRESS = 0x20  # Address of the 16inp device
+INPUTS16_INPORT_REG_ADD = 0  # Command to read the input bit
+pin16_mask = [
+    0x8000,
+    0x4000,
+    0x2000,
+    0x1000,
+    0x0800,
+    0x0400,
+    0x0200,
+    0x0100,
+    0x80,
+    0x40,
+    0x20,
+    0x10,
+    0x08,
+    0x04,
+    0x02,
+    0x01,
+]
+
+MEGA_DEVICE_ADDRESS = 0x50  # Address of the mega device
+
+MEGA_MEM_RELAY_SET = 1  # Command to set the relay active
+MEGA_MEM_RELAY_CLR = 2  # Command to set the relay inactive
+
+
+def write_channel(bus_number: int, stack: int, channel: int, value: int):
```

Comment:
Since you are type annotating, might as well annotate the return value:

```
def write_channel(bus_number: int, stack: int, channel: int, value: int) -> None:
```